code/modules/wiremod/integrated_circuit.dm

```diff
@@ -484,6 +485,10 @@
 /obj/item/integrated_circuit/proc/set_display_name(new_name)
 	display_name = new_name
 
+/obj/item/integrated_circuit/module/set_display_name(new_name)
+	..()
+	name = "module ([new_name])"
```

Comment:
what?

Reply:
Ah, i misunderstood some advice I got on how to rename only the modules. Should have tested that again.